rust-wasm-id-allocator/distributed-id-allocator/Cargo.toml

```diff
@@ -23,4 +23,5 @@ features = [
 ]
 
 [dependencies]
-id-types = { path = "../../rust-wasm-id-allocator/id-types", version = "0.1", default-features = false }
\ No newline at end of file
+id-types = { path = "../../rust-wasm-id-allocator/id-types", version = "0.1", default-features = false }
+avl = "0.7.1"
```

Comment:
The author's description for this crate gives a bit of pause:

> An ordered map and set implemented with an AVL tree (nearly balanced binary search tree) in Rust.

https://crates.io/crates/avl

Thoughts?

Reply:
It looks like this package currently has one published dependent. Is that a consideration for an acceptable dependency in production code?

Reply:
Ah, that's a bummer. That definitely gives me pause. That one dependent has < 100 downloads...so, might need to find an alternative :(